infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/SshKeysProvisioner.java

```diff
@@ -111,8 +123,31 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity identity)
     List<SshPairImpl> all = new ArrayList<>(vcsSshPairs);
     all.addAll(systemSshPairs);
 
-    doProvisionSshKeys(all, k8sEnv, workspaceId);
-    doProvisionVcsSshConfig(vcsSshPairs, k8sEnv, workspaceId);
+    List<String> invalidSshKeys =
+        all.stream()
+            .filter(keyPair -> !isValidSshKeyPair(keyPair))
+            .map(SshPairImpl::getName)
+            .collect(Collectors.toList());
+
+    if (!invalidSshKeys.isEmpty()) {
+      String message =
+          format(
+              Warnings.SSH_KEYS_WILL_NOT_BE_MOUNTED_MESSAGE,
+              invalidSshKeys.toString(),
+              identity.getWorkspaceId());
+      LOG.warn(message);
+      k8sEnv.addWarning(new WarningImpl(Warnings.SSH_KEYS_WILL_NOT_BE_MOUNTED, message));
+      runtimeEventsPublisher.sendRuntimeLogEvent(message, ZonedDateTime.now().toString(), identity);
```

Comment:
is this message somehow marked as warning? (Do we even have mechanism for warnings?) From the attached screenshot it's not clearly visible that something went wrong.

Reply:
I doubt. @sleshchenko raised this idea as well. I hope they will have some progress in the future.

Reply:
The new issue is created to make warnings used again https://github.com/eclipse/che/issues/19054